kerningview/src/main/java/com/andrognito/kerningview/KerningTextView.java

```diff
@@ -51,7 +51,8 @@ private void init(AttributeSet attrs, int defStyle) {
         try {
             kerningFactor = currentTypedArray.getFloat(R.styleable.KerningViews_kv_spacing,
                     NO_KERNING);
-            originalText = originalTypedArray.getText(0);
+            CharSequence attributeText = originalTypedArray.getText(0);
+            originalText = attributeText != null ? attributeText : "";
```

Comment:
Didn't call through to `setText` to avoid double call of `applyKerning`. But it may make sense to also guard the `setText` method in case `this` or an extending class calls `setText` with a `null` value (which  `TextView` won't)